.github/workflows/continuous-integration.yml

```diff
@@ -72,20 +72,6 @@ jobs:
         run: |
           python -m pip install --upgrade pip
 
-#      - name: Get pip cache dir
-#        id: pip-cache-dir
-#        run: |
-#          echo "::set-output name=dir::$(python -m pip cache dir)"
-#
-#      - name: pip cache
-#        uses: actions/cache@v2
-#        id: pip-cache
-#        with:
-#          path: ${{ steps.pip-cache-dir.outputs.dir }}
-#          key: ${{ matrix.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('**/requirements.txt') }}
-#          restore-keys: |
-#            ${{ matrix.os }}-${{ matrix.python-version }}-pip-
-
```

Comment:
Installing the Python dependencies takes about 2 minutes. Could it be beneficial to cache them too, as attempted here a long time ago?

Reply:
Actually, the "Install project" step also runs `pip` commands and takes about 2 minutes. The total time saving could be substantial.

Reply:
Last time I've tried this (16c4c5f20d) I've encountered a bunch of errors and hastily concluded that `pip`-caching is fragile. Perhaps at that time too many things were changing at once (_i.e._ new versions of CI runner and Python dependencies). Nevertheless, I also want faster CI tests so I'll try this again 😁